Libraries/Text/TextInput/Multiline/RCTMultilineTextInputView.m

```diff
@@ -48,6 +48,8 @@ - (instancetype)initWithBridge:(RCTBridge *)bridge
 #else // [TODO(macOS GH#774)
     _scrollView.documentView = _backedTextInputView;
     _scrollView.contentView.postsBoundsChangedNotifications = YES;
+    // NSTextView has no focus ring by default so let's use the standard Aqua focus ring.
+    _scrollView.focusRingType = NSFocusRingTypeExterior;
```

Comment:
Can we make this controlled by the 'enableFocusRing' prop?

Reply:
RCTUITextView doesn't have 'enableFocusRing' integrated like RCTUITextField. Let me try that though

Reply:
Thanks. I remember there were was an open source contribution to support that prop in single line text inputs I think, telling me it is a thing that people care about turning on and off. I'm iffy about going against Apple and adding a focus ring where there wasn't one natively, but a prop should be fine :)

Reply:
it depends. if it is like a huge canvas like notes.app i think they don't show it. but if it is like a field like apple's feedback app it does make sense to show focus ring.
![MicrosoftTeams-image](https://user-images.githubusercontent.com/20715435/180499436-6f683e69-7a9a-4407-8ed2-a30e8ee5adf4.png)

Reply:
More reason to support the prop!

Reply:
I tried adding 'enableFocusRing' to RCTUIScrollView just like the way RCTUITextField handles it, for some reason it isn't working for me

Reply:
@harrieshin Oh I meant NSTextView out of the box doesn't have focus ring enabled, unlike TextField

Reply:
I think I figured it out, I had to initialize the prop when scrollView gets instantiated.